### PR TITLE
Add a new quote

### PR DESCRIPTION
**Pull request by verv0eren**
I am a newbie learning to use the Github.

### CODE REVIEW

File: quotes [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"If you want to be happy, be." - Leo Tolstoy
```
Comment (hemanth) [inline quotes]: Why was this removed?

Comment (verv0eren) [inline quotes]: Sorry! I should have just added a new quote at the bottom. Initially, I thought of making a new version with all my chosen quotes but then I stalled and decided to just try and watch how pull requests work.

**Comment by hemanth** [comment]
Congrats on your first contribution.

**Comment by verv0eren** [comment]
Thank you!
